STYLEGUIDE.md

```diff
@@ -134,3 +134,15 @@ export enum CollisionType {
   Fixed = 'Fixed'
 }
 ```
+
+## Site and docs language
+
+#### DO
+
+ - Use "Excalibur" (vs. "excalibur", "excalibur.js", "excaliburjs") when referring to the project or the broader ecosystem
+  - e.g. "Excalibur has built in types"
+  - e.g. "The Excalibur community can be found on Discord"
+
+#### DON'T
+
+ - Avoid referencing "new" code. Instead the docs should represent the current point in time. This minimizes the docs appearing stale over time
```

Comment:
Might be a hot take, push back!

Reasoning: Throughout the docs I see Excalibur referred to differently (sometimes Excalibur.js, sometimes Excalibur, etc), and sometimes on the same page. Its minor, and some may not notice, but from a branding perspective I think the consistency is good. 

It does go against the word mark, which is Excalibur.js - but its not uncommon for the word mark / logo to be a bit different.

Reply:
I tend to write like I speak (terrible habbit)